adobecom/express
# QA guide widget in Sidekick

PR for the QA guide sidekick widget
This feature allows a streamlined human QA process with a checklist for each page in the QA flow. There are 2 pre-requisites to this feature that are a qa-guide.docx in the docs folder (or what's defined in QA_GUIDE_FILE_LOCATION in sidekick.js) and a QA Log defined in QA_LOG_FILE_LOCATION = '/express/qa-log' in qa-guide.js

The qa-guide.docx will use the qa-guide block. It could be one catch-all block or the user has the option to split it into desktop and mobile separate QA streams using the desktop or the mobile variant of the block.

Resolves: no ticket yet.

Test URLs:
- Before: https://stage--express--adobecom.hlx.page/express/learn/blog?martech=off
- After: https://qa-guide-widget--express--adobecom.hlx.page/express/learn/blog?martech=off

You might be able to change your sidekick settings in chrome to test this feature but I've found that the most reliable way to try it is to follow the steps below:
- check out the `sidekick-guided-qa` branch
- run hlx up locally
- go to https://sidekick-guided-qa--express--adobecom.hlx.live/express/
- find the QA guide button in the sidekick and click it

I'm avoiding using express/ as LHS testing target because we currently have LCP Identification issue.
